### PR TITLE
Translated your first component

### PR DESCRIPTION
**Pull request by Akshit1025**
I have translated the **Your First Component** page. Please review it

**Sections on the page**
> * [ ] Intro
> 
> * [ ] Components: UI Building Blocks
> 
> * [ ] Defining a Component
>   * [ ] Step 1: Export the component
>   * [ ] Step 2: Define the function
>   * [ ] Step 3: Add Markup
> 
> * [ ] Using a component
>   * [ ] What the browser sees
>   * [ ] Nesting and Organising Components
> 
> * [ ] Recap
> 
> * [ ] Challenge
>   * [ ] Export the Component
>   * [ ] Fix the return statement
>   * [ ] Spot the Mistake
>   * [ ] Your Own Component

Please let me know if there are any changes that have to be made
Please checkmark these boxes too
Thank You

### CODE REVIEW

**Comment by Akshit1025** [comment]
@arshadkazmi42 please review this pull request

Thank You

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42 please review this pull request
> 
> 
> 
> Thank You

@Akshit1025 I have been a bit busy. I will get to it soon. Sorry about the delay

**Comment by Akshit1025** [comment]
> > @arshadkazmi42 please review this pull request
> > Thank You
> 
> @Akshit1025 I have been a bit busy. I will get to it soon. Sorry about the delay

ok no problem

File: beta/src/pages/learn/your-first-component.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+- React एप्लिकेशन में कौम्पोनॅन्ट क्या भूमिका निभाते हैं
+- अपना पहला React कौम्पोनॅन्ट कैसे लिखें
 
 </YouWillLearn>
 
-## Components: UI building blocks {/*components-ui-building-blocks*/}
+## कौम्पोनॅन्ट: UI बिल्डिंग ब्लॉक्स {/* components-ui-building-blocks */}
 
-On the Web, HTML lets us create rich structured documents with its built-in set of tags like `<h1>` and `<li>`:
+Web पर, HTML हमें `<h1>` और `<li>` जैसे टैग के अंतर्निहित सेट के साथ समृद्ध संरचित दस्तावेज़ बनाने देता है:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:21]: `समृद्ध संरचित दस्तावेज़` => `रिच स्ट्रक्चर्ड डाक्यूमेंट्स`

hindi translation seems a bit complex. lets keep it in hinglish

File: beta/src/pages/learn/your-first-component.md [hunk 2]
```diff
@@ -31,11 +31,11 @@ On the Web, HTML lets us create rich structured documents with its built-in set
 </article>
 ```
 
-This markup represents this article `<article>`, its heading `<h1>`, and an (abbreviated) table of contents as an ordered list `<ol>`. Markup like this, combined with CSS for style, and JavaScript for interactivity, lies behind every sidebar, avatar, modal, dropdown—every piece of UI you see on the Web.
+यह मार्कअप इस लेख `<article>`, इसके शीर्षक `<h1>`, और एक (संक्षिप्त) सामग्री तालिका को एक क्रमबद्ध सूची `<ol>` के रूप में दर्शाता है। इस तरह का मार्कअप, स्टाइल के लिए CSS के साथ संयुक्त, और जावास्क्रिप्ट अन्तरक्रियाशीलता के लिए, प्रत्येक साइडबार, अवतार, मोडल, ड्रॉपडाउन के पीछे निहित है — UI का हर टुकड़ा जो आप Web पर देखते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:34]: `सामग्री तालिका` => `टेबल ऑफ़ कंटेंट्स`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:34]: `लेख` => `आर्टिकल`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:34]: `संयुक्त` => `मिला के`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:34]: `अन्तरक्रियाशीलता` => `इंटरएक्टिविटी`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:34]: `निहित` => `रहता`

File: beta/src/pages/learn/your-first-component.md [hunk 3]
```diff
@@ -31,11 +31,11 @@ On the Web, HTML lets us create rich structured documents with its built-in set
...
 
-React lets you combine your markup, CSS, and JavaScript into custom "components," **reusable UI elements for your app.** The table of contents code you saw above could be turned into a `<TableOfContents />` component you could render on every page. Under the hood, it still uses the same HTML tags like `<article>`, `<h1>`, etc.
+React आपको अपने मार्कअप, CSS और जावास्क्रिप्ट को कस्टम "कौम्पोनॅन्ट," **आपके एप्प के लिए पुन: प्रयोज्य `UI` तत्वों में संयोजित करने देता है। ** ऊपर देखी गई सामग्री कोड की तालिका को `<TableOfContents />` कौम्पोनॅन्ट में बदल दिया जा सकता है आप हर पेज पर रेंडर कर सकते हैं। हुड के तहत, यह अभी भी उसी HTML टैग का उपयोग करता है जैसे `<article>`, `<h1>`, आदि।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:36]: `**आपके एप्प के लिए पुन: प्रयोज्य `UI` तत्वों में संयोजित करने देता है। **` => `**आपके एप्प के लिए री-यूज़ब्ले UI एलिमेंट्स** में संयोजित करने देता है।`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:36]: `सामग्री कोड की तालिका` => `टेबल ऑफ़ कंटेंट्स कोड`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:36]: `है आप` => `है जो आप`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:36]:
```
हुड के तहत, यह अभी भी उसी HTML टैग का उपयोग करता है जैसे `<article>`, `<h1>`, आदि।
```
=>
```
अंडर हुड, यह अभी भी HTML टैग जैसे `<article>`, `<h1>`, आदि का उपयोग करता है।
```

File: beta/src/pages/learn/your-first-component.md [hunk 4]
```diff
@@ -31,11 +31,11 @@ On the Web, HTML lets us create rich structured documents with its built-in set
...
 
-Just like with HTML tags, you can compose, order and nest components to design whole pages. For example, the documentation page you're reading is made out of React components:
+HTML टैग्स की तरह ही, आप संपूर्ण पेज को डिज़ाइन करने के लिए कौम्पोनॅन्ट को कंपोज़, ऑर्डर और नेस्ट कर सकते हैं। उदाहरण के लिए, आप जिस दस्तावेज़ पेज को पढ़ रहे हैं वह React कौम्पोनॅन्ट से बना है:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:38]: `दस्तावेज़` => `डॉक्यूमेंटेशन`

File: beta/src/pages/learn/your-first-component.md [hunk 5]
```diff
@@ -51,58 +51,55 @@ Just like with HTML tags, you can compose, order and nest components to design w
 </PageLayout>
 ```
 
-As your project grows, you will notice that many of your designs can be composed by reusing components you already wrote, speeding up your development. Our table of contents above could be added to any screen with `<TableOfContents />`! You can even jumpstart your project with the thousands of components shared by the React open source community like [Chakra UI](https://chakra-ui.com/) and [Material UI](https://material-ui.com/).
+जैसे-जैसे आपकी परियोजना बढ़ती है, आप देखेंगे कि आपके कई डिज़ाइन आपके द्वारा पहले से लिखे गए कौम्पोनॅन्ट का पुन: उपयोग करके, आपके विकास को गति देकर बनाए जा सकते हैं। ऊपर दी गई हमारी सामग्री तालिका `<TableOfContents />` के साथ किसी भी स्क्रीन पर जोड़ी जा सकती है! आप React ओपन सोर्स समुदाय जैसे [Chakra UI](https://chakra-ui.com/) और [Material UI](https://material-ui.com/) द्वारा साझा किए गए हजारों कौम्पोनॅन्ट के साथ अपनी परियोजना को शुरू भी कर सकते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `आपकी परियोजना बढ़ती` => `आपका प्रोजेक्ट बढ़ता`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `आपके विकास को गति देकर बनाए जा सकते हैं।` => `आपके डेवलपमेंट की गति को बढ़ाते हैं।`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `दी गई हमारी सामग्री तालिका` => `दिए गए हमारे टेबल ऑफ़ कंटेंट्स`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `जोड़ी` => `ऐड किया जा सकता है`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `समुदाय` => `कम्युनिटी`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `साझा` => `शेयर`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `अपनी परियोजना` => `अपने प्रोजेक्ट`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:54]: `को शुरू भी कर` => `को एक शुरुआत दे`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on this.

I have added few comments till line 55

Lets' get these fixed and then I will continue with further review.

You can check for our review guidelines at #23

File: beta/src/pages/learn/your-first-component.md (hunk 2 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/your-first-component.md]: @Akshit1025 There is a typo here.

`सटेबल ऑफ़ कंटेंट्सा` => `टेबल ऑफ़ कंटेंट्स`

[the remark this answers is not in the document]
Comment (arshadkazmi42) [reply beta/src/pages/learn/your-first-component.md]: @Akshit1025 Typo here. Can you please check once

File: beta/src/pages/learn/your-first-component.md (hunk 3 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/your-first-component.md]: @Akshit1025 typo here

File: beta/src/pages/learn/your-first-component.md (hunk 5 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/your-first-component.md]: @Akshit1025 Typo here

**Comment by arshadkazmi42** [comment]
@Akshit1025 There are few typo. I have marked them as unresolved.

Can you cross check those and post that I will start reviewing the rest of the PR

**Comment by Akshit1025** [comment]
@arshadkazmi42 Hey sorry for responding really late but I have fixed the typos that you have asked me to do so please carry on with the reviewing of the file

Thanks

File: beta/src/pages/learn/your-first-component.md [hunk 6]
```diff
@@ -51,58 +51,55 @@ Just like with HTML tags, you can compose, order and nest components to design w
...
+}
 ```
 
 </Sandpack>
 
-And here's how to build a component:
+यहाँ एक कौम्पोनॅन्ट बनाने का तरीका बताया गया है:
 
-### Step 1: Export the component {/*step-1-export-the-component*/}
+### Step 1: कौम्पोनॅन्ट निर्यात करें {/* step-1-export-the-component */}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:78]: `निर्यात` => `एक्सपोर्ट`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:78]: `{/* step-1-export-the-component */}` => `{/*step-1-export-the-component*/}`

File: beta/src/pages/learn/your-first-component.md [hunk 7]
```diff
@@ -51,58 +51,55 @@ Just like with HTML tags, you can compose, order and nest components to design w
...
 
-The `export default` prefix is a [standard JavaScript syntax](https://developer.mozilla.org/docs/web/javascript/reference/statements/export) (not specific to React). It lets you mark the main function in a file so that you can later import it from other files. (More on importing in [Importing and Exporting Components](/learn/importing-and-exporting-components)!)
+`export default` उपसर्ग एक [standard JavaScript syntax](https://developer.mozilla.org/docs/web/javascript/reference/statements/export) है (React के लिए विशिष्ट नहीं)। यह आपको फ़ाइल में मुख्य फ़ंक्शन को चिह्नित करने देता है ताकि आप बाद में इसे अन्य फ़ाइलों से आयात कर सकें। ([Importing and Exporting Components](/learn/importing-and-exporting-components) में आयात करने पर अधिक!)
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `उपसर्ग` => `प्रीफिक्स`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `standard JavaScript syntax` => `स्टैण्डर्ड जावास्क्रिप्ट सिंटेक्स`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `विशिष्ट` => `स्पेसिफिक`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `मुख्य` => `मैन`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `चिह्नित` => `मार्क`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `आयात` => `इम्पोर्ट`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `Importing and Exporting Components` => `इम्पॉर्टिंग एंड एक्सपोर्टिंग कौम्पोनॅन्टस`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:80]: `आयात करने पर अधिक!` => `इम्पॉर्टिंग के बारे में और अधिक जानें!`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Added review comments till line 81. 

Let's get these fixed and then we can start the review of rest of the file.

**Comment by Akshit1025** [comment]
@arshadkazmi42 I have resolved the typos you have pointed out
please check out

sorry for responding late as I had school too

File: beta/src/pages/learn/your-first-component.md [hunk 8]
```diff
@@ -76,33 +76,33 @@ img { height: 200px; }
...
 
 <Gotcha>
 
-React components are regular JavaScript functions, but **their names must start with a capital letter** or they won't work!
+React कौम्पोनॅन्ट नियमित जावास्क्रिप्ट फ़ंक्शन हैं, लेकिन **उनके नाम बड़े अक्षर से शुरू होने चाहिए** या वे काम नहीं करेंगे!
 
 </Gotcha>
 
-### Step 3: Add markup {/*step-3-add-markup*/}
+### Step 3: मार्कअप जोड़ें {/*step-3-add-markup*/}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:95]: `जोड़ें` => `ऐड करें`

File: beta/src/pages/learn/your-first-component.md [hunk 9]
```diff
@@ -76,33 +76,33 @@ img { height: 200px; }
...
 
-The component returns an `<img />` tag with `src` and `alt` attributes. `<img />` is written like HTML, but it is actually JavaScript under the hood! This syntax is called [JSX](/learn/writing-markup-with-jsx), and it lets you embed markup inside JavaScript.
+कौम्पोनॅन्ट `src` और `alt` विशेषताओं के साथ एक `<img />` टैग देता है। `<img />` HTML की तरह लिखा गया है, लेकिन वास्तव में यह जावास्क्रिप्ट है! इस सिंटैक्स को [JSX](/learn/writing-markup-with-jsx) कहा जाता है, और यह आपको जावास्क्रिप्ट के अंदर मार्कअप एम्बेड करने देता है।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:97]: `टैग देता है` => `रिटर्न करता है`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on the fixes.

I have added more review comments till line 120.

Once these are fixed, we can continue further with the review.

**Comment by Akshit1025** [comment]
@arshadkazmi42 I have fixed the typos. You can now start reviewing them further

thanks

File: beta/src/pages/learn/your-first-component.md [hunk 10]
```diff
@@ -114,24 +114,19 @@ return (
 
 <Gotcha>
 
-Without parentheses, any code on the lines after `return` [will be ignored](https://stackoverflow.com/questions/2846283/what-are-the-rules-for-javascripts-automatic-semicolon-insertion-asi)!
+ब्रैकेट के बिना, `return` के बाद लाइनों पर लिखा कोई भी कोड [इग्नोर हो जाएगा](https://stackoverflow.com/questions/2846283/what-are-the-rules-for-javascripts-automatic-semicolon-insertion-asi)!
 
 </Gotcha>
 
-## Using a component {/*using-a-component*/}
+## एक कौम्पोनॅन्ट का उपयोग करना {/* using-a-component */}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:121]: `{/* using-a-component */}` => `{/*using-a-component*/}`

Let's not make any changes to these. These will be used for reflinks

File: beta/src/pages/learn/your-first-component.md [hunk 11]
```diff
@@ -114,24 +114,19 @@ return (
...
 
-Now that you've defined your `Profile` component, you can nest it inside other components. For example, you can export a `Gallery` component that uses multiple `Profile` components:
+अब जब आपने अपना `Profile` कौम्पोनॅन्ट परिभाषित कर लिया है, तो आप इसे अन्य कौम्पोनॅन्ट के अंदर घोंसला बना सकते हैं। उदाहरण के लिए, आप एक `Gallery` कौम्पोनॅन्ट निर्यात कर सकते हैं जो एकाधिक `Profile` कौम्पोनॅन्ट का उपयोग करता है:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:123]: `परिभाषित` => `डिफाइन`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:123]: `घोंसला बना` => `इस्तेमाल`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:123]: `निर्यात` => `एक्सपोर्ट`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:123]: `एकाधिक` => `एक से ज़्यादा`

File: beta/src/pages/learn/your-first-component.md [hunk 12]
```diff
@@ -114,24 +114,19 @@ return (
...
 
 ```js
 function Profile() {
-  return (
-    <img
-      src="https://i.imgur.com/MK3eW3As.jpg"
-      alt="Katherine Johnson"
-    />
-  );
+  return <img src="https://i.imgur.com/MK3eW3As.jpg" alt="Katherine Johnson" />;
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:129]: Let's not change any code alignments 

```
  return (
    <img
      src="https://i.imgur.com/MK3eW3As.jpg"
      alt="Katherine Johnson"
    />
  );
```

File: beta/src/pages/learn/your-first-component.md [hunk 13]
```diff
@@ -147,19 +142,22 @@ export default function Gallery() {
 ```
 
 ```css
-img { margin: 0 10px 10px 0; height: 90px; }
+img {
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:145]: same as above. lets not change any code alignments

```
img { margin: 0 10px 10px 0; height: 90px; }
```

File: beta/src/pages/learn/your-first-component.md [hunk 14]
```diff
@@ -147,19 +142,22 @@ export default function Gallery() {
...
+  margin: 0 10px 10px 0;
+  height: 90px;
+}
 ```
 
 </Sandpack>
 
-### What the browser sees {/*what-the-browser-sees*/}
+### ब्राउज़र क्या देखता है {/* what-the-browser-sees */}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:153]: `{/* what-the-browser-sees */}` => `{/*what-the-browser-sees*/}`

File: beta/src/pages/learn/your-first-component.md [hunk 15]
```diff
@@ -147,19 +142,22 @@ export default function Gallery() {
...
 
-Notice the difference in casing:
+आवरण में अंतर पर ध्यान दें:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:155]: `आवरण` => `केसिंग`

File: beta/src/pages/learn/your-first-component.md [hunk 16]
```diff
@@ -147,19 +142,22 @@ export default function Gallery() {
...
 
-* `<section>` is lowercase, so React knows we refer to an HTML tag.
-* `<Profile />` starts with a capital `P`, so React knows that we want to use our component called `Profile`.
+- `<section>` लोअरकेस है, इसलिए React जानता है कि हम एक HTML टैग को संदर्भित करते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:157]:
```
- `<section>`
```
=>
```
* `<section>`
```

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:157]: `संदर्भित` => `रेफेर`

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:157]: `करते` => `कर रहे`

File: beta/src/pages/learn/your-first-component.md [hunk 17]
```diff
@@ -147,19 +142,22 @@ export default function Gallery() {
...
+- `<Profile />` पूंजी `P` से शुरू होता है, इसलिए React जानता है कि हम `Profile` नामक हमारे कौम्पोनॅन्ट का उपयोग करना चाहते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:158]:
```
- `<Profile />`
```
=>
```
* `<Profile />`
```

Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:158]: `पूंजी` => `कैपिटल`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on the fixes.

I have added more review comments till line 170.

Let's get these fixed, post that we can continue further with the review

**Comment by Akshit1025** [comment]
@arshadkazmi42 I have fixed the typos and now you can further review the translation

File: beta/src/pages/learn/your-first-component.md (hunk 12 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/your-first-component.md]: This still has some spaces wrong. You can check the below screenshot

Screenshot from 2022-12-04 12-49-40

You can copy paste from the original version from [here](https://github.com/reactjs/hi.reactjs.org/pull/204/files#diff-987109509fdb76c4b375a50d1545ac66a321a4dbfb786d5fe75c84760e30b7fdL130-L134)

File: beta/src/pages/learn/your-first-component.md (hunk 13 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/your-first-component.md:150]: This is not fixed correctly. You can copy the original version from left side [here](https://github.com/reactjs/hi.reactjs.org/pull/204/files#diff-987109509fdb76c4b375a50d1545ac66a321a4dbfb786d5fe75c84760e30b7fdL150)

**Comment by arshadkazmi42** [comment]
@Akshit1025 There are two review comments which still have minor issues. Rest looks fine.

I have marked them as unresolved, can you check and update those? Post that we can continue further.

File: beta/src/pages/learn/your-first-component.md (hunk 12 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/your-first-component.md]: There is still one typo in this. The semicolon `;` is at wrong place

```
function Profile() {
  return (
    <img 
      src="https://i.imgur.com/MK3eW3As.jpg" 
      alt="Katherine Johnson" 
    />;
  )
}
```
=>
```
function Profile() {
  return (
    <img
      src="https://i.imgur.com/MK3eW3As.jpg"
      alt="Katherine Johnson"
    />
  );
}
```
```

**Comment by arshadkazmi42** [comment]
@Akshit1025 There is still one typo in the code block. Please check the unresolved comment

**Comment by Akshit1025** [comment]
@arshadkazmi42 typos have been fixed. Please carry on

File: beta/src/pages/learn/your-first-component.md [hunk 18]
```diff
@@ -389,22 +374,23 @@ export default function Gallery() {
 ```
 
 ```css
-img { margin: 0 10px 10px 0; }
+img {
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:377]:
```suggestion
img { margin: 0 10px 10px 0; }
```

File: beta/src/pages/learn/your-first-component.md [hunk 19]
```diff
@@ -389,22 +374,23 @@ export default function Gallery() {
...
+  margin: 0 10px 10px 0;
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:378]:
```suggestion
```

File: beta/src/pages/learn/your-first-component.md [hunk 20]
```diff
@@ -389,22 +374,23 @@ export default function Gallery() {
...
+}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:379]:
```suggestion
```

File: beta/src/pages/learn/your-first-component.md [hunk 21]
```diff
@@ -389,22 +374,23 @@ export default function Gallery() {
...
 
 </Sandpack>
 
 </Solution>
 
-### Your own component {/*your-own-component*/}
+### आपका अपना कौम्पोनॅन्ट {/*your-own-component*/}
 
-Write a component from scratch. You can give it any valid name and return any markup. If you're out of ideas, you can write a `Congratulations` component thats shows `<h1>Good job!</h1>`. Don't forget to export it!
+Scratch से एक कौम्पोनॅन्ट लिखें। आप इसे कोई भी मान्य नाम दे सकते हैं और कोई मार्कअप वापस कर सकते हैं। यदि आपके पास विचार नहीं हैं, तो आप एक `Congratulations` कौम्पोनॅन्ट लिख सकते हैं जो `<h1>Good job!</h1>` दिखाता है। इसे निर्यात करना न भूलें!
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:388]:
```suggestion
बिलकुल शुरू से एक कौम्पोनॅन्ट लिखें। आप इसे कोई भी वैलिड नाम दे सकते हैं और कोई मार्कअप रिटर्न कर सकते हैं। यदि आपके पास कोई आईडिया नहीं हैं, तो आप एक `Congratulations` कौम्पोनॅन्ट लिख सकते हैं जो `<h1>Good job!</h1>` दिखाता है। इसे एक्सपोर्ट करना न भूलें!
```

File: beta/src/pages/learn/your-first-component.md [hunk 22]
```diff
@@ -415,14 +401,12 @@ Write a component from scratch. You can give it any valid name and return any ma
 
 ```js
 export default function Congratulations() {
-  return (
-    <h1>Good job!</h1>
-  );
+  return <h1>Good job!</h1>;
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/your-first-component.md:404]:
```suggestion
  return (
    <h1>Good job!</h1>
  );
```

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on the fixes.

I have added review in rest of the file.

Once these are fixed we can get this merged.

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on the fixes.

I have added review in rest of the file.

Once these are fixed we can get this merged.

**Comment by Akshit1025** [comment]
@arshadkazmi42 all the typos that you have suggested have been completed

**Review by arshadkazmi42 — APPROVED** [review]
Great work. Thank you for working on all the fixes.

lets get this merged :tada:

**Pull request merged**